index.js

```diff
@@ -121,22 +122,33 @@ export default class VideoPlayer extends Component {
     this.onSeekRelease = this.onSeekRelease.bind(this);
     this.onSeek = this.onSeek.bind(this);
     this.onSeekByPosition = this.onSeekByPosition.bind(this);
+    this.activateStatistics = this.activateStatistics.bind(this);
+    this.deactivateStatistics = this.deactivateStatistics.bind(this);
   }
 
   componentDidMount() {
     if (this.props.autoplay) {
       this.hideControls();
+      this.props.stats && this.setState({ stats: true })
     }
   }
 
   componentWillUnmount() {
-    stats.deactivate()
+    this.deactivateStatistics()
     if (this.controlsTimeout) {
       clearTimeout(this.controlsTimeout);
       this.controlsTimeout = null;
     }
   }
 
+  deactivateStatistics () {
+    this.state.stats && this.deactivateStatistics()
```

Comment:
@8ntexerra  are you sure about this? I suggest here is should be stats.deactivate method